### DISCUSSION TITLE
when i move a repo to a list , i want a filter to hide it

### DISCUSSION THREAD

**Discussion by Takanashikoucha** [community/community#8690] [Other Feature Feedback, Questions, & Ideas] [answered]
maybe it can called "in list" or something else

**Comment by brianlovin** [comment, marked as answer]
Thanks for the suggestion @Takanashikoucha — I've added this to our tracking issue, as several others have asked for similar behavior 😊

**Reply by egypjoy** [reply]
Thank stressing

**Reply by Kilobites-cmd** [reply]
- [ ]

**Comment by Avivhdr** [comment]
Hi @brianlovin 👋🏼 
This is an amazing new feature, love it 🤩 

But I'm surprised that this suggestion doesn't have more 👍🏼.

When I started categorizing my starred, I felt this imidietly.

My thought on the same issue was to change the UI from this:
![image](https://user-images.githubusercontent.com/23265867/145707329-ad06d701-d4f7-45c2-95d7-97310f32a47e.png)

To this:
 
![image](https://user-images.githubusercontent.com/23265867/145707399-f04951f4-a439-4210-a3b5-77541e12e0e6.png)

I believe that this is better because:
1. Let's say that I have 200 starred and now I want to categorize all of them. Currently, I don't have any way of knowing which ones were already moved to their list and which ones were not.
2. At some point in the future my I will have many lists and with the current UI, the long list of lists pushed the search component way down.

**Reply by brianlovin** [reply]
Totally, this all makes sense — we've had a handful of other discussions created from people requesting what you've mocked up here, I'll make sure the team takes a look to see what's possible!

**Reply by Big297** [reply]
Big297

**Comment by awlobo** [comment]
Does someone know what's the status on this? I would like to have the possibility to hide repos that are already inside a list, so that it is easier to know which ones are not contained in any list.

**Reply by Big297** [reply]
Big297

**Reply by tamtura75** [reply]
![bunny-color.png](https://user-images.githubusercontent.com/96757090/199382069-4a813987-3550-4291-9769-8e780c5709d4.png)

**Reply by tamtura75** [reply]
- - - [ ] ~~@tamtura75 ~~

**Comment by Kilobites-cmd** [comment]
> maybe it can called "in list" or something else
